pyproject.toml

```diff
@@ -46,6 +46,10 @@ pyright = "^1.1.294"
 ruff = "^0.0.291"
 types-SQLAlchemy = "^1.4.31"
 
+[tool.poetry.group.dev.dependencies]
+surrogate = "^0.1"
```

Comment:
This repo is not very well supported - not saying we shouldn't use it - but perhaps we should fork it into `climatepoicyradar` ?

Reply:
I'll raise a ticket for this.